doc/developers/advanced_installation.rst

```diff
@@ -274,9 +282,8 @@ scikit-learn from source:
 
 .. prompt:: bash $
 
-    conda create -n sklearn-dev -c conda-forge python numpy scipy cython \
-        joblib threadpoolctl pytest compilers llvm-openmp
-    conda activate sklearn-dev
+    conda env create -f environments/common.yml
```

Comment:
I don't know if we could come up with something similar to the requirements of scikit-image but for conda.